webui/react/src/components/LogViewer/LogViewerFilters.tsx

```diff
@@ -30,12 +29,12 @@ export interface Filters {
 export const ARIA_LABEL_RESET = 'Reset';
 
 export const LABELS: Record<keyof Filters, string> = {
-  agentIds: 'Agent',
-  allocationIds: 'Allocation',
-  containerIds: 'Container',
-  levels: 'Level',
-  rankIds: 'Rank',
-  searchText: 'Search',
+  agentIds: 'Agents',
+  allocationIds: 'Allocations',
+  containerIds: 'Containers',
+  levels: 'Levels',
+  rankIds: 'Ranks',
+  searchText: 'Searchs',
```

Comment:
nit: I don't see this actually being used anywhere, but it should probably be "Searches"

Reply:
oh yes. thanks